modules/mri_violations/README.md

```diff
@@ -0,0 +1,86 @@
+# MRI violations
+
+## Purpose
+
+The MRI violations module provides a front end to view information
+about scans that were flagged as violations of the imaging protocol
+(which is configurable and set by each study) by the imaging pipeline 
+scripts.
+
+## Intended Users
+
+The MRI violations module is primarily used by DCC staff and the MR 
+Imaging Committee who both decide whether a scan should be included 
+in the batch of study scans or, alternatively, if it should be excluded 
+from further analysis. The violations also serve as prompts to inform 
+scanner technologists that settings need to be adjusted for future 
+scans (i.e. number of slices are being reduced from the protocol), or 
+as a cue that something is wrong (i.e. coil not functioning properly).
+
+
+## Scope
+
+The MRI violations module shows a summary of the MRI violations
+(one image per row) identified by the imaging pipeline scripts.
+Depending on the type of violation, clicking on the scan's problem
+will redirect the user to pages with more information regarding
+the identified violation. The different types of violation are:
+
+- Could not identify scan type
+- Protocol Violation
+- Candidate Mismatch
+
+Additionally, the violations "Resolution Status" column can be
+updated once the violation has been resolved. Available options
+are:
+
+- **Unresolved**: this indicates this scan still needs to be 
+addressed by the DCC or MRI committee
+- **Reran**: this indicates that this scan will be rerun through 
+the imaging insertion pipeline and inserted
+- **Emailed site/pending**: this indicates that the site has been 
+emailed about this scan, and a resolution or answer is pending 
+(i.e.. mismatched PSCID and DCCID in the naming convention or a visit 
+label that has not yet been created)
+- **Inserted**: this means that the scan has been inserted into 
+the Imaging Browser, and no flag was needed
+- **Rejected**: this means that the scan has been excluded for future 
+analysis and will not be inserted into the Imaging Browser nor 
+available for download from the DQT
+- **Inserted with flag**: this means that the scan has been inserted 
+into the Imaging Browser and a caveat flag was attached - this caveat 
+flag should have either a drop down menu or open text for the person 
+approving the insertion to indicate what the flag is for
+- **Other**: open text??
```

Comment:
I think this one still needs a bit of work..

Reply:
@driusan yeah... I am not sure what to do with this one. Even Leigh does not seem to know what is that option. :S Any ideas what we should do?

Reply:
"Other: resolutions that don't fit in any of the above categories"?

I'm not sure what "open text" means.. *is* there an open text field to describe it?

Reply:
@driusan Great! Thanks! Just made the last change. Let me know if you notice something else.